builder/query/sql.go

```diff
@@ -6,10 +6,12 @@ import (
 
 type SQL struct {
 	Base
-	Query        string         `json:"query,omitempty"`
-	ResultFormat string         `json:"resultFormat,omitempty"`
-	Header       *bool          `json:"header,omitempty"`
-	Parameters   []SQLParameter `json:"parameters,omitempty"`
+	Query          string         `json:"query,omitempty"`
+	ResultFormat   string         `json:"resultFormat,omitempty"`
+	Header         *bool          `json:"header,omitempty"`
+	TypesHeader    *bool          `json:"typesHeader,omitempty"`
+	SqlTypesHeader *bool          `json:"sqlTypesHeader,omitempty"`
```

Comment:
nitpick: probably this should be SQLTypesHeader instead since Golang suggests to use capital letters for acronyms ( https://go.dev/talks/2014/names.slide#5 )